### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by DenielWorld** [korado531m7/CustomRecipe#1]
The plugin doesn't work when registering a recipe.

**Comment by korado531m7** [comment]
yup, because i'm not supported on latest version of pocketmine. to use this, i need to update.

**Comment by DenielWorld** [comment]
Then update xD

**Comment by korado531m7** [comment]
if i'm free, ill do it.
but please note that these days, i'm so busy.
so can't that now :'(
